src/lib.rs

```diff
@@ -523,17 +540,20 @@ fn set_ec_point_encoding(val: config::EcPointEncoding) -> CK_RV {
 }
 
 #[cfg(all(test, feature = "fips", feature = "nssdb"))]
-fn get_fips_behavior(save: &mut config::FipsBehavior) -> CK_RV {
-    let gconf = global_rlock!(noinitcheck CONFIG);
-    *save = gconf.conf.fips_behavior.clone();
+fn get_fips_behavior(
```

Comment:
These should probably get a doc strings. It looks like there is more like that missing doc strings in this file.

Reply:
Oh good catch, thanks

Reply:
The State structure seem to be missing doc strings entirely, if you are ok with it I will open a separate PR to document it.

Reply:
-> #243